src/common/attributes/useCommonUserAttributes.js

```diff
@@ -125,6 +125,14 @@ export default (t) => useMemo(() => ({
     name: t('attributeWebLiveRouteLength'),
     type: 'number',
   },
+  'web.lineWidth': {
```

Comment:
I think it's better to call it `mapLineWidth`. Adding `web.` is an old way. Usually now we just have a camel-case name without dots.